server/service/src/vaccination/get_vaccination_card.rs

```diff
@@ -18,8 +18,19 @@ pub struct VaccinationCard {
 pub struct VaccinationCardItem {
     pub row: VaccinationCardRow,
     pub suggested_date: Option<NaiveDate>,
+    pub status: Option<VaccinationCardItemStatus>,
 }
 
+#[derive(Clone, Debug)]
+pub enum VaccinationCardItemStatus {
+    Given,
+    NotGiven,
+    Pending,
+    Late,
+}
+
+const MONTHS_PER_YEAR: f64 = 365.25 / 12.0;
```

Comment:
Since we're re-using it. Do we have a global place to store constants we might want to refer to elsewhere?

Reply:
yes! `server/util/src/constants.rs` 🔥

Reply:
This is `DAYS_PER_MONTH` though right?